src/Veldrid/BlendStateDescription.cs

```diff
@@ -45,6 +45,22 @@ public BlendStateDescription(RgbaFloat blendFactor, params BlendAttachmentDescri
             AttachmentStates = new BlendAttachmentDescription[] { BlendAttachmentDescription.AlphaBlend }
         };
 
+        /// <summary>
+        /// Describes a blend state in which a single color target is blended with <see cref="BlendAttachmentDescription.AdditiveBlend"/>.
+        /// </summary>
+        public static readonly BlendStateDescription SingleAdditiveBlend = new BlendStateDescription
+        {
+            AttachmentStates = new BlendAttachmentDescription[] { BlendAttachmentDescription.AdditiveBlend }
+        };
+
+        /// <summary>
+        /// Describes a blend state in which a single color target is blended with <see cref="BlendAttachmentDescription.Disabled"/>.
```

Comment:
I'm not sure about the naming of this one, or the awkward comment (nothing is really "blended"). Suggestions welcome...

Reply:
I think it's fine, and makes sense. We probably wouldn't want to leave "Single" out, since the number of attachments is important. I doubt we'll have anything other than "Single" versions of these helpers, but the info is still important enough to keep in the name, I think.

The rest of the wording is good in that it is consistent with the other descriptions.